grafana-plugin/src/utils/consts.ts

```diff
@@ -54,3 +54,5 @@ export enum PAGE {
 }
 
 export const TEXT_ELLIPSIS_CLASS = 'overflow-child';
+
+export const INCIDENT_HORIZONTAL_SCROLLING_STORAGE = 'isIncidentalTableHorizontalScrolling';
```

Comment:
Constant for storing the key (whether horizontal scrolling is enabled or not) in local storage. Can be moved to some specific filename if we add more I guess